From 0c7c562482575cacaecadcd7913ef25aeb21711f Mon Sep 17 00:00:00 2001
From: Simon Ser <contact@emersion.fr>
Date: Tue, 10 Mar 2020 14:54:48 +0100
Subject: [PATCH 13/14] Update version to 0.10.1

---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 9096d43f..0056b8a9 100644
--- a/meson.build
+++ b/meson.build
@@ -1,7 +1,7 @@
 project(
 	'wlroots',
 	'c',
-	version: '0.10.0',
+	version: '0.10.1',
 	license: 'MIT',
 	meson_version: '>=0.51.0',
 	default_options: [
-- 
2.26.0

